index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prem Chand's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer"
+  />
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+</head>
+<body>
+    <header class = "pb-2">
+        <nav class = "pt-3 pb-3">
+            <ul class = "unordered-list d-flex flex-row justify-content-between">
+                <li class = "my-logo-line mt-1"><a href="">My Logo</a></li>
```

Comment:
Great job on implementing the changes requested by the previous reviewer. However, there are still some areas where the colors can be further improved to match the design more accurately. I recommend carefully reviewing the design guidelines and making the necessary adjustments to ensure consistency. you are using color `#007bff` but as per design it should be `#6070FF`. kindly update your code. 🙏 

Remember, paying attention to details like color can greatly enhance the overall visual appeal and user experience of your project. Keep up the good work and continue striving for design accuracy!

Reply:
for my-logo-line i was using color #6070FF. I had not used #007bff.